x-pack/platform/plugins/shared/alerting/public/pages/maintenance_windows/components/create_maintenance_windows_form.tsx

```diff
@@ -207,56 +201,33 @@ export const CreateMaintenanceWindowForm = React.memo<CreateMaintenanceWindowFor
     onSubmit: submitMaintenanceWindow,
   });
 
-  const [{ recurring, timezone, solutionId, startDate, endDate }, _, mounted] =
-    useFormData<FormProps>({
-      form,
-      watch: ['recurring', 'timezone', 'solutionId', 'scopedQuery', 'startDate', 'endDate'],
-    });
+  const [{ recurring, timezone, startDate, endDate }, _, mounted] = useFormData<FormProps>({
```

Comment:
You still use `solutionId` in `submitMaintenanceWindow`, won't this possibly lead to issues of it not being updated?

Reply:
removed everywhere